Lib/ufoLib/glifLib.py

```diff
@@ -242,10 +242,11 @@ def getGLIF(self, glyphName):
 			needRead = True
 		if needRead:
 			fileName = self.contents[glyphName]
-			if not os.path.exists(path):
+			try:
+				with open(path, "rb") as f:
+					text = f.read()
+			except FileNotFoundError:
```

Comment:
`FileNotFoundError` does not exist on python 2, use `IOError`

Reply:
though if you wanted to be sure it's a "file not found" you'd need to check the `errno` but, life is too short